playbooks/preflight-check.yml

```diff
@@ -175,3 +175,80 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1372 Check NFS exsiting
+- name: Check Glance NFS backend on glance
+  hosts: glance_all
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Check mount on Glance
+      shell: |
+        awk '/nfs/' /proc/mounts
+      register: nfs_mount_found
+
+    - name: Fail when Glance has NFS mount
+      fail:
+        msg: "Check failed: Glance currently has NFS mount."
+      when: nfs_mount_found.stdout
+
+    - name: Glance default store in api config
+      shell: |
+        awk '/default_store.*file/' /etc/glance/glance-api.conf
+      register: default_store_glance
+
+    - name: Fail when Glance default store is NFS
+      fail:
+        msg: "Check failed: Glance default store is NFS."
+      when: default_store_glance.stdout
+
+- name: Check Glance NFS config on infa hosts
+  hosts: "infra_hosts:os-infra_hosts"
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Config file list
+      shell: |
+        find /etc/openstack_deploy /etc/openstack_deploy/conf.d -type f -iname user_*.yml
+      register: config_file_list
+
+    - name: List config file has glance_default_store
+      shell: |
+        awk '/glance_default_store.*file/' "{{ item }}"
+      register: glance_default_store_config
+      with_items:
+        - "{{ config_file_list.stdout_lines }}"
```

Comment:
This is obsolete as we check as we already look at the override "glance_default_store"